folly/test/TryTest.cpp

```diff
@@ -132,7 +132,7 @@ TEST(Try, ValueOverloads) {
 // Make sure we can copy Trys for copyable types
 TEST(Try, copy) {
   Try<int> t;
-  auto t2 = t;
+  const auto& t2 = t;
```

Comment:
This is wrong and changes what the test does.

Reply:
Very valid. Seems like, in general, we wouldn't care much about clang-tidy over unit test code as higher risk for less value-add.